HatemHashlamon/task_manager.py

```diff
@@ -0,0 +1,119 @@
+from typing import List
+from task import *
+from utils import *
+
+# List to store all tasks (including urgent tasks)
+tasks: List[Task] = []
+
+def display_menu():
+    print("\nTask Manager")
+    print("1. Add Task")
+    print("2. View Tasks")
+    print("3. Update Task")
+    print("4. Delete Task")
+    print("5. Exit")
+
+def add_task():
+    try:
+        task_id = len(tasks) + 1
```

Comment:
This method of getting the ID is invalid. Remember that we have a delete operation. I also prefer to have a dictionary instead of a list, with the task ID as the key and the task as the value. Additionally, you can use UUID (yesterday we posted a Blog for that) as an ID for the task, and it should be in the Task class.